xml/System.Collections.Generic/Dictionary`2.xml

```diff
@@ -750,9 +750,9 @@
   
  This code example is part of a larger example provided for the <xref:System.Collections.Generic.Dictionary%602> class.  
   
- [!code-cpp[Generic.Dictionary#2](~/samples/snippets/cpp/VS_Snippets_CLR/Generic.Dictionary/cpp/source.cpp#2)]
- [!code-csharp-interactive[Generic.Dictionary#2](~/samples/snippets/csharp/VS_Snippets_CLR/Generic.Dictionary/CS/source.cs#2)]
- [!code-vb[Generic.Dictionary#2](~/samples/snippets/visualbasic/VS_Snippets_CLR/Generic.Dictionary/VB/source.vb#2)]  
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR/Generic.Dictionary/cpp/source.cpp" id="Snippet2":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Generic.Dictionary/CS/source.cs" id="Snippet2":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR/Generic.Dictionary/VB/source.vb" id="Snippet2":::
   
  ]]></format>
         </remarks>
```

Comment:
This file is still missing the interactive for https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.system-collections-idictionary-add

The API you're looking for is down at line 2208
https://github.com/dotnet/dotnet-api-docs/blob/ce1eaa66c6214b4e8ed52ecc26c4e5cd94295b13/xml/System.Collections.Generic/Dictionary%602.xml#L2208

Reply:
okay let me update this

Reply:
Done, let me know if this looks good :), apologies for missing this one.